apollo-client/src/main/java/com/ctrip/framework/apollo/util/yaml/YamlParser.java

```diff
@@ -25,12 +27,14 @@
 public class YamlParser {
   private static final Logger logger = LoggerFactory.getLogger(YamlParser.class);
 
+  private PropertiesFactory propertiesFactory= ApolloInjector.getInstance(PropertiesFactory.class);
```

Comment:
please format this line too

Reply:
there are some more lines with the similar issue, please also check and formar them